src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs

```diff
@@ -281,6 +285,11 @@ private void CachedValueChanged()
             }));
         }
 
+        private void ViewModel_RequestAutoCompletePopupPlacementTarget(Popup popup)
+        {
+            popup.PlacementTarget = this;
```

Comment:
👍

Reply:
I guess this way is better than `PlacementRectangle` since it is regardless of zoom level